pom.xml

```diff
@@ -64,7 +64,7 @@
     <properties>
         <junit.version>4.12</junit.version>
         <geronimo-activation.version>1.1</geronimo-activation.version>
-        <geronimo-javamail.version>1.8.3</geronimo-javamail.version>
+        <geronimo-javamail.version>1.8.4</geronimo-javamail.version>
```

Comment:
In James we switched to sun javax.mail

Shouldn't we do the same here?

Cc @mbaechler @rouazana ?